1/contrib/s2i/run

```diff
@@ -12,6 +12,21 @@ source /usr/local/bin/kube-slave-common.sh
 jenkins_config_dir="/opt/openshift/configuration"
 jenkins_config_path="${jenkins_config_dir}/config.xml"
 
+
+CONTAINER_MEMORY_IN_BYTES=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
+DEFAULT_MEMORY_CEILING=$((2**30-1))
+if [ "${CONTAINER_MEMORY_IN_BYTES}" -lt "${DEFAULT_MEMORY_CEILING}" ]; then
```

Comment:
so isn't this effectively saying you can never set a memory limit higher than 1gig?  (if you set your memory limit to 1.5gigs, then no max heap value gets set)

(where 2**30==1gig if i've done my math correctly)